### PR TITLE
fix: add createBucket file and updated relevant test

### PR DESCRIPTION
**Pull request by sofisl**
Fix: internal reference b/144088253

### CODE REVIEW

**Comment by sofisl** [comment]
@fhinkel - I'm not sure why the build test isn't passing. It seems to be an issue with the storage.ts test under system-test, but I didn't touch that test, and confirmed that it's up-to-date with the master branch. Not sure if I'm missing something or if it's a bug...

**Comment by fhinkel** [comment]
@sofisl I don't see right away why the system test is failing. But there are hundreds of tests, I wonder if the new test is somehow messing with the system tests, because it is creating a bucket and not deleting it. 

Can you delete the existing `createBucket` code and test in the sample. And we should change the new test to delete the bucket at the end of the test run. Otherwise we're left with a bucket after running the test once.

**Comment by bcoe** [comment]
@sofisl @fhinkel I have a hunch that this is a flaky test, just kicked off the build to hopefully confirm.

**Comment by fhinkel** [comment]
Don't we need to delete the existing `createBucket` sample? I only see code additions.

File: samples/createNewBucket.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  const storage = new Storage();
+
+  async function createBucket() {
+    // Creates a new bucket in the Asia region with the coldline default storage
+    // class. Leave the second argument blank for default settings.
+    //
+    // For default values see: https://cloud.google.com/storage/docs/locations and
+    // https://cloud.google.com/storage/docs/storage-classes
+
+    await storage.createBucket(bucketName, {
```
Comment (stephenplusplus) [inline samples/createNewBucket.js]: Should we show that this returns a Bucket object?

```js
const [bucket] = await storage.createBucket(bucketName, {
  location: 'ASIA',
  storageClass: 'COLDLINE',
});
```

Comment (sofisl) [reply samples/createNewBucket.js]: Great catch! Fixed both issues. Thanks for the review!

**Comment by sofisl** [comment]
@fhinkel - Lint is failing because I don't reassign the variable bucket, but that's the way the docs and @stephenplusplus recommend doing it.

Error:

Screen Shot 2019-11-25 at 12 50 06 PM

Also, I think the kokoro system test is flaky, according to @bcoe.

**Comment by sofisl** [comment]
@fhinkel - I think the system test is still flaky, this is the error - https://screenshot.googleplex.com/MZ27aYXMaeD

Otherwise all the tests have passed

**Comment by sofisl** [comment]
@fhinkel - waiting on CL to be approved to merge cl 284587412

**Comment by fhinkel** [comment]
@sofisl Sorry I didn't see this. Always better to ping me on chat rather than relying on GitHub notifications. LGTM! Thanks

**Pull request merged**